build-system/tasks/karma.conf.js

```diff
@@ -40,7 +40,17 @@ module.exports = {
   browserify: {
     watch: true,
     debug: true,
-    transform: ['babelify'],
+    transform: process.env.TRAVIS ? ['babelify'] : [
```

Comment:
This is causing mangling of imported src files when debugging tests in Karma. We should only do this for the coverage tool.